package.json

```diff
@@ -25,8 +25,7 @@
       "screwdriver-config-parser",
       "screwdriver-data-schema",
       "screwdriver-datastore-sequelize",
-      "screwdriver-executor-k8s",
```

Comment:
You'll need to add these to the Dockerfile, or there will be no executor modules to require.